boa_zksync/compiler_utils.py

```diff
@@ -73,3 +73,17 @@ def detect_expr_type(source_code, contract):
             except InvalidType:
                 pass
     return None
+
+
+def get_compiler_output(output):
+    # we need this helper method to get the correct key containing compiler output
+    # from the compiler. Assuming key names could change and also assuming that the
+    # number of keys could change, this method breaks if any of that happens:
+    
+    excluded_keys = {"version", "zk_version"}
+    contract_keys = set(output.keys()) - excluded_keys
+    
+    if len(contract_keys) != 1:
+        raise ValueError(f"Expected exactly one contract key, found {len(contract_keys)}")
```

Comment:
Error could be a bit more informative
```suggestion
        raise ValueError(f"Expected exactly one contract key, found {contract_keys}")
```